operator/config/crd/kustomization.yaml

```diff
@@ -10,4 +10,5 @@ resources:
 - bases/forklift.konveyor.io_plans.yaml
 - bases/forklift.konveyor.io_providers.yaml
 - bases/forklift.konveyor.io_storagemaps.yaml
+- bases/forklift.konveyor.io_ovirtimageiopopulators.yaml
```

Comment:
I need to add somehow the CRD of `VolumePopulator` (from [volume-data-source-validator](https://github.com/kubernetes-csi/volume-data-source-validator)). but it seems it requires also to add it into the pkg/api, is it? i would like to somehow apply it without it. @mnecas

Reply:
that's one option we talked about yesterday, or to extend the path that controller-gen looks at